app/code/Magento/CustomerGraphQl/Model/Customer/IsEmailAvailableDataProvider.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\CustomerGraphQl\Model\Customer;
+
+use Magento\Customer\Api\AccountManagementInterface;
+
+/**
+ * Is Customer Email Available checker
+ */
+class IsEmailAvailableDataProvider
```

Comment:
Hello @VitaliyBoyko,
Looks like in this case we don't need a class wrapper 
we can call `AccountManagementInterface` directly in our resolver
Thanks